lighthouse-core/lib/traces/tracing-processor.js

```diff
@@ -9,7 +9,8 @@
 // first frame of the response.
 const BASE_RESPONSE_LATENCY = 16;
 const SCHEDULABLE_TASK_TITLE = 'TaskQueueManager::ProcessTaskFromWorkQueue';
-const SCHEDULABLE_TASK_TITLE_ALT = 'ThreadControllerImpl::DoWork';
+const SCHEDULABLE_TASK_TITLE_ALT1 = 'ThreadControllerImpl::DoWork';
```

Comment:
can you add a comment pointing to https://crrev.com/c/883346

Reply:
done